assets/shiny-server.js

```diff
@@ -362,22 +544,22 @@
 
     //var relURL = getRelativePath(parentURL, this.url)
     
-    this.conn.send(formatOpenEvent(this.id, this.url));
+    this._owner.send(formatOpenEvent(this.id, this.url));
     this.onopen();
   };
   MultiplexClientChannel.prototype.send = function(data) {
     if (this.readyState === 0)
       throw new Error("Invalid state: can't send when readyState is 0");
     if (this.readyState === 1)
-      this.conn.send(formatMessage(this.id, data));
+      this._owner.send(formatMessage(this.id, data));
   };
   MultiplexClientChannel.prototype.close = function(code, reason) {
     if (this.readyState >= 2)
       return;
     debug("Close channel " + this.id);
-    if (this.conn.readyState === 1) {
+    if (this._owner.getConn().readyState === 1) {
```

Comment:
`readyState` is already handled (and buffered while disconnected) within `_owner.send`. Why are you bypassing that here (and in `send` above)?